cas/service.c

```diff
@@ -20,6 +20,7 @@
  */
 
 
+#include "dtm0/tx_desc.h"
```

Comment:
(style) usually, we add "include" directives after M0_TRACE_SUBSYSTEM definition. Let's add it to the group of the other dtm0/* directives. Also, please check this thing is needed: I guess dtm0/fop.h should include it.

Reply:
Moved it along with other dtm0 directives as suggested.
We still need to add dtm0/tx_desc.h separately because the coding guidelines say that we should explicitly include the headers that we need and not depend on it being included by others.

Reply:
If are just using exported structures and functions, there isn't any more symbol isolation to be achieved.
I would agree, however, to keep the list of include directives short and I will remove this include.